data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,280 @@ event "remnant: pantica void sprites"
+			`	"We have loaded the research laboratory onboard. The team will install it if they can find the space, but if they do not, it will be secured in the cargo bay," The Remnant scientist explains. "Before you depart, please verify that the laboratory is installed in the correct location. You know your ship better that we do, after all." She turns back to Dawn and makes a gesture of respect. "Embers guide your path, Seeker." With that, she strides away into the hive of activity that is the outfitting bays.`
+			choice
+				`	"So, what is our next step?"`
+					goto labnext
+				`	"Seeker?"`
+			`	Dawn looks a little embarassed. "Seeker is a title of respect for deep space explorers. It is unofficial - just a token of respect - but it is a title that I both aspire to earn and do not feel I deserve yet."`
+			choice
+				`	"I thought all Remnant were spacefarers?"`
+				`	"Okay. So what next?"`
+					goto labnext
+			`	"Oh, we are. Every Remnant has had some experience in space. But there is a big difference between a few weeks of duty in a spacefaring post and spending months or years drifting in the void surrounding distant systems."`
+			choice
+				`	"Years drifting in the void?"`
+				`	"Okay, let's get on with this."`
+					goto labnext
+			`	"If you want good data, you need to spend time to get it. And especially when we are tracking stellar phenomenon, you need periods of long observation. Snapshots of the sky are good - that is what we will be collecting on this trip. But for watching the galaxy at large, other civilizations, and things of that ilk... Nothing is better than long-term observation." She looks lost in thought for a moment. "That is neither here nor there, however.`
+			label labnext
+			`	"Now that you have the research laboratory, you will need to locate a jump drive in order to reach Sagittarius A*." She pauses and checks her commlink. "If you do not already have one, you can salvage one from a Korath Raider. Our early warning system indicates that there are several incoming Korath fleets that should be here within the next day or two. Any one of them should have a jump drive you can take. Meet me in the starport once you have a jump drive."`
+
+
+
+event "remnant: lab availability"
+	outfitter "Remnant"
+		add "Research Laboratory"
+
+
+
+mission "Remnant: Expanded Horizons Astral 2"
+	name "Research expedition to Sagittarius A*"
+	description "Travel to Sagittarius A* with a research laboratory installed in order to make measurements on the supermassive black hole and then return to <destination>."
+	blocked "You need at least 1 free bunk, a jump drive, and a research laboratory to take this mission."
```

Comment:
I don't see the requirement for a free bunk nor the passengers keyword.

Reply:
Never mind.  I see that the lab requires 1 crew.

Reply:
Hmmm... Right now, Dawn is taking the place of the crew member that would go with the research laboratory, so this is actually an oversight. (There used to be both an extra crew member & Dawn). So I'll remove that bunk mention.